app/src/renderer/connectors/lcdClient.js

```diff
@@ -28,9 +23,7 @@ const Client = (axios, localLcdURL, remoteLcdURL) => {
       if (Array.isArray(args)) {
         args = args.join(`/`)
       }
-      if (method === `DELETE`) {
```

Comment:
I remember that the DELETE requests needed to have this nested data structure. Did you test if any DELETE requests works like this?

Reply:
No, I tried to but I'm unable to start Voyager on this branch.

I deleted the above code because the behavior changed when I called Axios using `axios({method: 'delete', ...})` instead of `axios.delete(...` and this change was necessary to make the tests pass again.  At least one of the tests was the contract test `keys.delete` against a real HTTP server so it's likely that any other delete requests will work this way as well.

Reply:
cool, just wanted to make sure